src/main/src/test/java/org/geoserver/test/GeoServerSystemTestSupport.java

```diff
@@ -1579,11 +1593,12 @@ protected MockHttpServletResponse dispatch(HttpServletRequest request) throws Ex
 
     protected MockHttpServletResponse dispatch(HttpServletRequest request, String charset)
             throws Exception {
-        if (charset == null) {
-            charset = Charset.defaultCharset().name();
-        }
+
         MockHttpServletResponse response = new MockHttpServletResponse();
-        response.setCharacterEncoding(charset);
+
+        if(charset != null) {
+            response.setCharacterEncoding(charset); //if this is not set, it falls back on a default
+        }
```

Comment:
Formatting (space on if, comment before the line, not at its end)

Reply:
Done